docs/markdown/ios/avatar.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Avatar
+description: Avatar is used to represent a user. Every Avatar image has a subtle color wash.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/30/7a/de/307adebcacf4fca359de788e6a077329.png" alt="avatar hero image"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
+      - To reflect a person, company, or brand within the product
```

Comment:
super nit: as much as it physically pains me, we don't use Oxford commas in our style guide for the docs (and all of Pinterest) 😭